tests/browserstack_automation/specs/DiscussPage.js

```diff
@@ -25,45 +25,45 @@ describe('Discuss Page', () => {
   // // });
 
   // Discuss_004
-  it('verifyEmailVerificationButton', async () => {
+  it('verifyvoterEmailAddressVerificationButton', async () => {
     await DiscussPage.load();
     await driver.switchWindow('https://quality.wevote.us/news');
-    await expect(DiscussPage.emailVerificationButton).not.toBeClickable();
-    const element = await DiscussPage.emailTextBox; // Locate the text box element using a selector
+    await expect(DiscussPage.voterEmailAddressVerificationButton).not.toBeClickable();
+    const element = await DiscussPage.enterVoterEmailAddressTextBox; // Locate the text box element using a selector
     element.setValue('wevote@gmail.com');
-    await driver.waitUntil(async () => ((DiscussPage.toggleEmailVerificationButton)));
+    await driver.waitUntil(async () => ((DiscussPage.togglevoterEmailAddressVerificationButton)));
```

Comment:
In a future pull request, please fix the text "togglevoter" to camel-case "toggleVoter".